### ISSUE TITLE
Deleted posts should be visible for favorite groups

### ISSUE THREAD

**Issue by BrokenEagle** [danbooru/danbooru#4347]
This was brought up on Discord by @fredgido.

It makes sense to hide deleted posts in pools, but in favorite groups it doesn't as a user may be trying to group posts together to illustrate something, such as my own [favgroup #2573](https://danbooru.donmai.us/favorite_groups/2573) (**Not found IQDB**).

In addition to showing posts in the show view, the **Posts** link should also have `status:any` added on as well.

So to sum up:

- [ ] Show all deleted posts in the favorite groups show view
- [ ] Have the **Posts** link on the favgroup show view include `status:any`
